exercises/practice/resistor-color-trio/.meta/example.rb

```diff
@@ -27,6 +27,12 @@ def value
   end
 
   def to_s
-    value < 1000 ? "#{value} ohms" : "#{(value.to_f/1000).to_i} kiloohms"
+    index = 0
+    value_copy = value
+    while value_copy >= 1000 && index < LABELS.size - 1
+      value_copy /= 1000
+      index += 1
+    end
+    "#{value_copy} #{LABELS[index]}"
```

Comment:
Trying to understand why the example files are changed here... 

I thought this was about templates, and example files are not generally given by the upstream.

Reply:
Yeah likely, I could still split it if you like?